static/cars_style.css

```diff
@@ -39,3 +40,11 @@ h3 {
     margin-right: auto;
     display: block;
 }
+
+.recall_info {
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
```

Comment:
@pengxuemoua Can we change this to `display: table;`

This has all of the items connecting to their bullet points. My centered view was very distorted